frontend/public/components/resource-quota.jsx

```diff
@@ -35,64 +55,125 @@ const Row = ({obj: rq}) => <div className="row co-resource-list__item">
+  const chartResourceTypes = ['requests.cpu', 'limits.cpu', 'requests.memory', 'limits.memory'];
+  return _.intersection(resourceTypes, chartResourceTypes).length > 0;
+};
+
+const Details = ({obj: rq}) => {
+  const resourceTypes = getResourceTypes(rq);
+  const showChartRow = showCharts(resourceTypes);
+  const popover = (
+    <Popover>
+      Requests are the amount of resources you expect to use. These are used when establishing if the cluster can fulfill your Request.<br />
+      <br />
+      Limits are a maximum amount of a resource you can consume. Applications consuming more than the Limit may be terminated.<br />
+      <br />
+      A cluster administrator can establish limits on both the amount you can Request and your Limits with a &nbsp;
+      <a href={helpLink('dev_guide/compute_resources.html#dev-compute-resources')} target="_blank" rel="noopener">Resource Quota</a>.
+    </Popover>
+  );
+  return <React.Fragment>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Resource Quota Overview" />
+      {showChartRow && <div className="co-resource-quota-chart-row">
+        {_.includes(resourceTypes, 'requests.cpu') &&
+          <div className="co-resource-quota-chart">
+            <Gauge title="CPU Request" percentUsed={getProportions(rq, 'requests.cpu').percentUsed} />
+          </div>}
+        {_.includes(resourceTypes, 'limits.cpu') &&
+        <div className="co-resource-quota-chart">
+          <Gauge title="CPU Limit" percentUsed={getProportions(rq, 'limits.cpu').percentUsed} />
+        </div>}
+        {_.includes(resourceTypes, 'requests.memory') &&
+        <div className="co-resource-quota-chart">
+          <Gauge title="Memory Request" percentUsed={getProportions(rq, 'requests.memory').percentUsed} />
+        </div>}
+        {_.includes(resourceTypes, 'limits.memory') &&
+        <div className="co-resource-quota-chart">
+          <Gauge title="Memory Limit" percentUsed={getProportions(rq, 'limits.memory').percentUsed} />
+        </div>}
+      </div>}
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={rq} podSelector="spec.podSelector" showNodeSelector={false} />
+        </div>
+        <div className="col-sm-6">
+          <dl className="co-m-pane__details">
+            <dt>Scopes</dt>
+            <QuotaScopes quota={rq} />
+          </dl>
+        </div>
       </div>
-      <div className="co-m-table-grid__body">
-        {getResourceTypes(rq).map(type => <ResourceUsageRow key={type} quota={rq} resourceType={type} />)}
+    </div>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Resource Quota Details" style={{display: 'block', marginBottom: '20px'}}>
+        <OverlayTrigger
+          overlay={popover}
+          placement="top"
+          trigger="click"
+          rootClose="true"
+        >
+          <span className="pficon pficon-info co-resource-quota-infotip" aria-hidden="true"></span>
```

Comment:
We should either add an `sr-only` label or make the entire overlay trigger hidden to screen readers.